src/vue/backend/src/views/GuidedSetupConnectIDXBrokerAccount.vue

```diff
@@ -0,0 +1,49 @@
+<template>
+    <GuidedSetupContentCard class="gs" cardTitle="Connect Your IDX Broker Account" :steps="steps" :relatedLinks="links">
```

Comment:
Will this class be used all the time when using the guided set up content card? If so, you can add it to the base component